packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -19,6 +19,7 @@
 #include "lwe/PlatformIntegrationData.h"
 #include "webview_factory.h"
 
+#define BUFFER_POOL_SIZE 5
```

Comment:
Why did you declare it to 5?

Reply:
It is not the value set this time, it is the value it already had.
There was no particular reason.
It was moderately selected for hands greater than 3. 
at multiful webviews, if frequent updates such as animations occur, buffer shortages may occur, so a moderately large value was chosen.

Reply:
I understood. Thanks for the details. 👍